types/member_store.ts

```diff
@@ -0,0 +1,19 @@
+export interface DISPATCH {
```

Comment:
having types folder will create a problem as types and pages get added the length of the folder will start increasing and it turns out to be bad to navigate.
instead what we can do here is we can have types inside the members folder itself with the naming convention of 
`members_store.type.ts ` now this will be easier to navigate as well as work on when we want to change and manage things